matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/Matrix.kt

```diff
@@ -61,85 +64,70 @@ class Matrix private constructor(context: Context, matrixConfiguration: MatrixCo
     @Inject internal lateinit var lightweightSettingsStorage: LightweightSettingsStorage
 
     init {
-        Monarchy.init(context)
-        DaggerMatrixComponent.factory().create(context, matrixConfiguration).inject(this)
-        if (context.applicationContext !is Configuration.Provider) {
+        val appContext = context.applicationContext
+        Monarchy.init(appContext)
+        DaggerMatrixComponent.factory().create(appContext, matrixConfiguration).inject(this)
+        if (appContext !is Configuration.Provider) {
             val configuration = Configuration.Builder()
                     .setExecutor(Executors.newCachedThreadPool())
                     .setWorkerFactory(matrixWorkerFactory)
                     .build()
-            WorkManager.initialize(context, configuration)
+            WorkManager.initialize(appContext, configuration)
         }
         ProcessLifecycleOwner.get().lifecycle.addObserver(backgroundDetectionObserver)
     }
 
+    /**
+     * Return the User Agent used for any request that the SDK is making to the homeserver.
+     * There is no way to change the user agent at the moment
+     */
     fun getUserAgent() = userAgentHolder.userAgent
 
+    /**
+     * Return the AuthenticationService
+     */
     fun authenticationService() = authenticationService
```

Comment:
not for this PR, we have a mix of `get${object}` and `${object}`, is it something we want to align on in the public api?

Reply:
Yes, we have a mix, the (un-official!) rule so far is to not use the `get` prefix for the SDK services, and I think this is the case for all services, to avoid having long name for the methods.

Happy to discuss this more, and/or to change the rule.

I have renamed `workerFactory()` because this is not a SDK service.